server/src/services/TeamScript.ts

```diff
@@ -70,15 +71,19 @@ export class TeamScript implements IService {
 
     @Action
     @Description("A cool way to post an issue to Teams using adaptive cards")
-    postIssue(): void
+    async postIssue(): Promise<void>
     {
         if (!this._outgoing) return;
 
-        fetch(this._outgoing, {
-            method: 'POST',
-            body: JSON.stringify(AreaCards.issueFormat(this._authorName, this._authorImage, this._repository, this._repositoryLink, this._issueId, this._issue, this._issueLink, this._message, this._datetime)),
-            headers: {'Content-Type': 'application/json'} 
-        }).then();
+        for (let issue of this._issues.issues)
+        {
+            await fetch(this._outgoing, {
+                method: 'POST',
+                body: JSON.stringify(AreaCards.issueFormat(issue.author, issue.authorImage, this._repository, issue.repoLink, issue.number, issue.title, issue.body, issue.issueLink, issue.createdAt)),
+                headers: {'Content-Type': 'application/json'} 
+            }).then().catch(e => console.error(e));
```

Comment:
I still don't get why there is `await` **AND** `.then()` around the same promise.
If it works then it's fine ig but if you have some explanation i would gladly hear it

Reply:
look better; i don't know what you're talking about :eyes:

Reply:
IDK what are you talking about